### ISSUE TITLE
fix(EstimatorReport): sklearn metrics are not labelled correctly in resulting DataFrame

### ISSUE THREAD

**Issue by auguste-probabl** [probabl-ai/skore#1714]
### Describe the bug

See reproducer

### Steps/Code to Reproduce

```python
from sklearn.datasets import load_breast_cancer
from sklearn.linear_model import LogisticRegression
from skore import train_test_split
from skore import EstimatorReport
X, y = load_breast_cancer(return_X_y=True)
split_data = train_test_split(X=X, y=y, random_state=0, as_dict=True)
classifier = LogisticRegression(max_iter=10_000)
report = EstimatorReport(classifier, **split_data)
report.metrics.report_metrics(
     scoring=["neg_log_loss"],
     indicator_favorability=True
)
```

### Expected Behavior

The index of the resulting DataFrame should read "Negative Log Loss", or perhaps "neg_log_loss".

### Actual Behavior

```python
          LogisticRegression Favorability
Metric
Log Loss            0.106768         (↘︎)
```

### Environment

```shell
skore 9bbfdb3
```

**Comment by glemaitre** [comment]
I actually think it is fine. The `neg_` particle is to show that is loss. If we really want to make it a score and thus label it `Negative Log Loss`, then, we need to negate it and turn the favorability. But I don't think we should do that.

We should expose the metric in the most natural way.

**Comment by glemaitre** [comment]
But users are not interested by the negative log loss in 100% of the cases, it is just that the scikit-learn API is annoying and crafted for score comparison/selection. For instance, there is no way for a scikit-learn user to have a scorer name that provide the positive log loss.
